components/InvestmentCard.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useState } from "react";
+import { TouchableOpacity, View, StyleSheet, Text } from "react-native";
+
+export default function InvestmentCard(props) {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>{props.fund}</Text>
+      <Text style={styles.text}>{props.risk}</Text>
+      <Text style={styles.amount}>{props.amount}</Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
We like to use tools that sort objects - It's often easier to read etc. Consider using: https://medium.com/@azat_io/eslint-plugin-for-sorting-objects-imports-types-enums-jsx-props-etc-f159efb778cf#:~:text=Perfectionist%20offers%20three%20types%20of,of%20which%20can%20be%20customized.